WordPress/src/main/java/org/wordpress/android/ui/prefs/LearnMorePreference.java

```diff
@@ -213,11 +213,11 @@ public SavedState createFromParcel(Parcel in) {
     }
 
     private class LearnMoreClient extends WebViewClient {
-        @Override
-        public boolean shouldOverrideUrlLoading(WebView webView, String url) {
-            // prevent loading clicked links
-            return !mUrl.equals(url) && !SUPPORT_CONTENT_JS.equals(url) && !CONTENT_PADDING_JS.equals(url);
-        }
+//        @Override
+//        public boolean shouldOverrideUrlLoading(WebView webView, String url) {
+//            // prevent loading clicked links
+//            return !mUrl.equals(url) && !SUPPORT_CONTENT_JS.equals(url) && !CONTENT_PADDING_JS.equals(url);
+//        }
```

Comment:
A couple of thoughts.  

We prefer to just remove code that is no longer needed rather than comment it out. It keeps things nice and tidy that way.

Removing this overridden method may have some side effects we probably don't want. If you check the git blame and history you can see where the [code was added](https://github.com/wordpress-mobile/WordPress-Android/pull/3754) and the [issue it was trying to solve](https://github.com/wordpress-mobile/WordPress-Android/issues/3678). 

 Maybe do a little more investigation to see how the issue could be addressed without removing this method.

Reply:
Done. It was a URL redirect of https://en.support.wordpress.com/settings/discussion-settings/#default-article-settings to https://wordpress.com/support/settings/discussion-settings/#default-article-setting that will trigger the calling of the method. So I changed the URL in site_settings.xml to https://wordpress.com/support/settings/discussion-settings/#default-article-settings instead and the issue is fixed